linopy/io.py

```diff
@@ -307,6 +307,93 @@ def to_file(m, fn, integer_label="general"):
     return fn
 
 
+def to_mosekpy(model, task=None):
+    """
+    Export model to MOSEK.
+
+    Export the model directly to MOSEK without writing files.
+
+    Parameters
+    ----------
+    m : linopy.Model
+    task : empty MOSEK task
+
+    Returns
+    -------
+    task : MOSEK Task object
+    """
+
+    import mosek
+
+    if task is None:
+        task = mosek.Task()
+
+    task.appendvars(model.nvars)
+    task.appendcons(model.ncons)
+
+    M = model.matrices
+    for j, n in enumerate(("x" + M.vlabels.astype(str).astype(object))):
+        task.putvarname(j, n)
```

Comment:
A pity that mosekpy does not support parallelized assignments. This will take quite long for large problems, but it seems that's the way to go.